src/js/themes/base.js

```diff
@@ -879,14 +877,24 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       size: `${baseSpacing}px`,
     },
     rangeInput: {
+      // extend: undefined
       track: {
         height: '4px',
-        color: css`
-          ${props => rgba(normalizeColor('border', props.theme), 0.2)};
-        `,
+        color: 'border',
+        // opacity: undefined,
```

Comment:
Should this be 0.2 to preserve the prior style?

Reply:
I don't think so. 
Assume the user changed the color to `color: 'brand'`, and we will apply an opacity of 0.2 on that color, this will be a breaking change for users that have modified the theme color property.
On StyledRangeInput I added a backward compatibility section that will take the opacity into account. this is why the snapshots are still intact.